can_tools/scrapers/official/SC/sc_demographics.py

```diff
@@ -39,9 +39,9 @@ def fetch(self):
 
         data = []
         for county in counties:
+            workbook = engine.setFilter("Recipient County for maps", county)
```

Comment:
don't need to set the county for each race demographic, only when switching counties, so I've moved this outside the inner loop